samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -325,6 +329,8 @@ spec:
         volumeMounts:
         - name: tmp
           mountPath: /tmp
+        securityContext:
```

Comment:
can you also add the same for the following?
reviews-v1
reviews-v2
reviews-v3

Reply:
Good point, added the `securityContext` to all app containers.